as-controller-board-status/res/configuration.toml

```
@@ -62,10 +62,10 @@ Type="edgex-messagebus"
 
 [ApplicationSettings]
   AverageTemperatureMeasurementDuration = "-15s"
-  DeviceName = "ds-controller-board"
+  DeviceName = "controller-board"
   MaxTemperatureThreshold = "83"
   MinTemperatureThreshold = "10"
-  MQTTEndpoint = "http://localhost:48082/api/v2/device/name/Inference-MQTT-device/command/inferenceDoorStatus"
+  MQTTEndpoint = "http://localhost:59882/api/v2/device/name/Inference-MQTT-device/inferenceDoorStatus"
```

Can we change the name of this setting? Name is confusing with what the endpoint is.
```suggestion
  DoorStatusCommandEndpoint = "http://localhost:59882/api/v2/device/name/Inference-MQTT-device/inferenceDoorStatus"
```